automated-comments: add initial disapprove comments for java/two-fer (#1120)
exercism/website-copy: automated-comments/java/two-fer/use_one_return.md

> This solution uses multiple return statements, but it should be simple to write

Reviewer: ```suggestion
This solution uses multiple return statements, but it is possible to write
```

> this solution with one return. Consider using a ternary operator to handle the

Reviewer: Add a link to ternary operator

> null input case.

Reviewer: ```suggestion
`null` input case.
```
